nginx/lib.sh

```
@@ -1,62 +1,15 @@
-RESOLVEEOF
-  nginx -t -c /tmp/resolve-check.conf >/dev/null 2>&1
-}
-
-render_full_conf() {
+render() {
   cat "$FRAGMENTS_DIR/00-redirect.conf.template" \
-      "$FRAGMENTS_DIR/10-app.conf.template" \
-      "$FRAGMENTS_DIR/20-dev-app.conf.template" > "$CONF_OUT"
-  envsubst '${DOMAIN} ${DEV_DOMAIN}' < "$CONF_OUT" > "${CONF_OUT}.rendered"
+      "$FRAGMENTS_DIR/${APP_FRAGMENT}" > "${CONF_OUT}.tmp" &&
+  envsubst '${DOMAIN} ${DEV_DOMAIN}' < "${CONF_OUT}.tmp" > "${CONF_OUT}.rendered" &&
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)